server/services/errorHandler.js

```diff
@@ -0,0 +1,30 @@
+module.exports = (err, req, res, next) => {
+  const error = {
```

Comment:
Is there any purpose why we have object in object? Why not just const err = { code: ... } ?